packages/design-system/src/ui/components/value-transformer/value-transformer.tsx

```diff
@@ -141,6 +141,7 @@ export function ValueTransformer({
   }, [transformers, setValue]);
 
   return React.cloneElement(children, {
+    // @ts-expect-error - React.HTMLAttributes is not typed
```

Comment:
I think this was always wrong but is now actually showing because I updated the type definitions for react